src/app/profiles/profile-detail/profile-detail.component.html

```diff
@@ -31,7 +31,27 @@
               </mat-option>
             </mat-select>
           </mat-form-field>
-          <div fxFlex="50" style="margin-top:15px">
+          <div fxFlex="100">
+            <mat-label>Redirection</mat-label>
```

Comment:
Change "Redirection" to "AMT Features" and move down above checkboxes.  Might also add a description that says something like "Enable/Disable redirection features of AMT" for bit more clarity.